src/PackageCompiler.jl

```diff
@@ -619,6 +640,14 @@ function create_app(package_dir::String,
     bundle_julia_libraries(app_dir)
     bundle_artifacts(ctx, app_dir)
 
+    # Load all packages in the Project into the sysimg for Apps. Without specifying this,
+    # we cannot serialize any methods from those packages, because the packages won't be
+    # loaded in the final julia process. Since we cannot know which packages will be needed
+    # we can load _all_ of the packages (and their recursive packages) from the project.
+    # This may be more than the user will actually need, but that is up to users to prune
+    # their Project files to only those `deps` actually in use.
+    packages = Symbol.(keys(Pkg.TOML.parsefile(ctx.env.project_file)["deps"]))
```

Comment:
```
julia> ctx.env.project.deps
Dict{String,Base.UUID} with 37 entries:
  "BinaryBuilder"           => UUID("12aac903-9f7c-5d81-afc2-d9565ea332ae")
  "Documenter"              => UUID("e30172f5-a6a5-5a46-863b-614d45cd2de4")
  "ForwardDiff"             => UUID("f6369f11-7733-5829-9624-2563aa707210")
...
```

is probably better

Reply:
```suggestion
    packages = keys(ctx.env.project.deps )
```
Awesome thanks. Does this look good? ⬆️

Reply:
Oops, I guess that should be an array of symbols:
```julia
    packages = Symbol.(keys(ctx.env.project.deps))
```
Fixed in https://github.com/JuliaLang/PackageCompiler.jl/pull/463/commits/af09bd8b7db975d818366998d889439b20141e2f